sympy/integrals/rubi/patterns.py

```diff
@@ -0,0 +1,2424 @@
+from sympy.external import import_module
+matchpy = import_module("matchpy")
+
+if matchpy:
+    Wildcard, Pattern, ReplacementRule, ManyToOneReplacer = matchpy.Wildcard, matchpy.Pattern, matchpy.ReplacementRule, matchpy.ManyToOneReplacer
```

Comment:
No, there is an old test which is failing code quality test for `patterns.py` file (only in python 2.7):

https://travis-ci.org/sympy/sympy/jobs/254914808